pkg/inject/inject.go

```diff
@@ -413,7 +413,7 @@ func (conf *ResourceConfig) injectPodSpec(patch *Patch, identity k8s.TLSIdentity
 			{Name: "LINKERD2_PROXY_LOG", Value: conf.proxyConfig.GetLogLevel().GetLevel()},
 			{
 				Name:  "LINKERD2_PROXY_CONTROL_URL",
-				Value: fmt.Sprintf("tcp://%s:%d", controlPlaneDNS, conf.proxyConfig.GetDestinationApiPort().GetPort()),
+				Value: fmt.Sprintf("tcp://%s:%d", controlPlaneDNS, 8086),
```

Comment:
Let's add a constant at the top of this file, near where `defaultKeepaliveMs` is defined, to hold this port value. I'm just a bit worried about it getting lost in the configuration code. Something like:

```diff
diff --git a/pkg/inject/inject.go b/pkg/inject/inject.go
index 938a6e20..9f40a0aa 100644
--- a/pkg/inject/inject.go
+++ b/pkg/inject/inject.go
@@ -31,6 +31,8 @@ const (
 	podNamespaceEnvVarName = "LINKERD2_PROXY_POD_NAMESPACE"
 	// defaultKeepaliveMs is used in the proxy configuration for remote connections
 	defaultKeepaliveMs = 10000
+	// destinationAPIPort is the port exposed by the linkerd-destination service
+	destinationAPIPort = 8086
 )
 
 var injectableKinds = []string{
@@ -420,7 +422,7 @@ func (conf *ResourceConfig) injectPodSpec(patch *Patch, identity k8s.TLSIdentity
 			{Name: "LINKERD2_PROXY_LOG", Value: conf.proxyConfig.GetLogLevel().GetLevel()},
 			{
 				Name:  "LINKERD2_PROXY_CONTROL_URL",
-				Value: fmt.Sprintf("tcp://%s:%d", controlPlaneDNS, 8086),
+				Value: fmt.Sprintf("tcp://%s:%d", controlPlaneDNS, destinationAPIPort),
 			},
 			{Name: "LINKERD2_PROXY_CONTROL_LISTENER", Value: fmt.Sprintf("tcp://0.0.0.0:%d", conf.proxyConfig.GetControlPort().GetPort())},
 			{Name: "LINKERD2_PROXY_METRICS_LISTENER", Value: fmt.Sprintf("tcp://0.0.0.0:%d", conf.proxyConfig.GetMetricsPort().GetPort())},
```

Reply:
I see this conversation marked as resolved, but I don't actually see the new constant in the diff. Is it possible this change didn't get pushed?

Reply:
Oops, forgot to add the file.
Should be 👍  now